### PR TITLE
fix: disable shuffling in distributed validation sampler

### PR DESCRIPTION
**Pull request by andrewwhitecdw**
### Problem
fix: disable shuffling in distributed validation sampler

### Fix
Replace:
```
    if torch.distributed.is_initialized():
        val_sampler = torch.utils.data.distributed.DistributedSampler(val_dataset)
    else:
        val_sampler = None
```

with:
```
    if torch.distributed.is_initialized():
        val_sampler = torch.utils.data.distributed.DistributedSampler(val_dataset, shuffle=False)
    else:
        val_sampler = None
```

### Files changed
- `data/dataloaders.py`

### CODE REVIEW

**Comment by andrewwhitecdw** [comment]
Closing this sweep-generated PR: sole commit is missing a valid Signed-off-by trailer. It does not meet the sweep requirements (single signed-off commit).
